### PR TITLE
add ladspa-sdk, move ladspaH

### PR DESCRIPTION
**Pull request by magnetophon**
ladspaH is part of ladspa-sdk, not ladspaPlugins.

ladspaPlugins is conventionally called swh-plugins, but changing that would brake backwards compatibility.
Maybe we should have both, "linked" via all-packages.nix?

### CODE REVIEW

File: pkgs/applications/audio/ladspa-sdk/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  };
+
+  patchPhase = ''
+    cd src
+    sed -i 's@/usr/@$(out)/@g'  makefile
+    sed -i 's@-mkdirhier@mkdir -p@g'  makefile
+  '';
+
+  meta = {
+    description = "The LADSPA SDK";
```
Comment (jagajaga) [inline pkgs/applications/audio/ladspa-sdk/default.nix]: This is not a good description.

File: pkgs/applications/audio/ladspa-sdk/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+stdenv.mkDerivation rec {
+  name = "ladspa_sdk_${version}";
```
Comment (jagajaga) [inline pkgs/applications/audio/ladspa-sdk/default.nix]: Change `_` to `-`

**Comment by magnetophon** [comment]
@jagajaga done.
What do you think about a rename of ladspaPlugins?
I'm working on an updated version now, so it'd be a good moment...

**Comment by jagajaga** [comment]
I think it's ok and don't affect a thing :)

**Comment by jagajaga** [comment]
Maybe add plugins in this PR?

**Pull request merged**